panou/src/App.tsx

```diff
@@ -1,17 +1,28 @@
 import React from "react";
-// import "./App.css";
-// import Registration from "./components/Registration/Registration";
-import Login from "./components/Login/Login";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import AppProvider from "./context/AppProvider";
 // import Modal from "./components/Modal/Modal";
+import PaginaPrincipala from "./components/HomePage/HomePage";
+import Registration from "./components/Registration/Registration";
+import Login from "./components/Login/Login";
+import NavBar from "./components/Navigation/NavBar/NavBar";
+import Container from "./components/Container/Container";
 
 function App() {
```

Comment:
sar putea de scris si in asa mod asta

plus la asta uite ca la componente functionale avem si typescript-ul ```React.FC```
```export const App: React.FC = () => (```